site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupImportStep2.html

```diff
@@ -41,6 +41,7 @@ <h4 rsf:id="newmemberheading:">import2.newmemberheading</h4>
            
 			<p class="act">
                 <input type="submit" rsf:id="continue" id="continue" value="Create Groups" onclick="SPNR.disableControlsAndSpin( this, null );" />
+				<input type="button" accesskey ="b" rsf:id="back" id="back" value="Back" onclick="history.back();" />
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.